static/app/views/dashboards/widgets/timeSeriesWidget/plottables/bars.tsx

```diff
@@ -34,8 +34,7 @@ export class Bars extends ContinuousTimeSeries<BarsConfig> implements Plottable
     return [
       BarSeries({
         name: timeSeries.field,
-        color,
-        stack: config.stack ?? GLOBAL_STACK_NAME,
+        stack: config.stack,
```

Comment:
Stack used to be required, because bar charts had all-or-nothing stacking. Now, stacking can be fully configurable, so we don't have a use for this fallback.